.github/workflows/run-tests.yml

```diff
@@ -46,6 +46,10 @@ jobs:
 
       - uses: arduino/setup-protoc@v3
 
+      - run: GOOS=windows GOARCH=amd64 make bins
+
+      - run: GOOS=darwin GOARCH=arm64 make bins
```

Comment:
I think this needs a `make clean-bins` since once the binary from the first `make bins` exists, the second one will not be invoked anymore (just tried this locally).

Reply:
ah, thanks for catching this. I didn't check file creation time when testing locally, so missed that the output files were not regenerated